### ISSUE TITLE
Get latest Grafana version has a error (json).

### ISSUE THREAD

**Issue by archr** [StackPointCloud/docker-influxdb#4]
Error with grafana_url:
```sh
Step 4 : RUN grafana_url=$(curl http://grafanarel.s3.amazonaws.com/latest.json | python -c 'import sys, json; print json.load(sys.stdin)["url"]') &&   curl -s -o grafana.tar.gz $grafana_url &&   curl -s -o influxdb_latest_amd64.deb http://s3.amazonaws.com/influxdb/influxdb_latest_amd64.deb &&   mkdir grafana &&   tar -xzf grafana.tar.gz --directory grafana --strip-components=1 &&   dpkg -i influxdb_latest_amd64.deb &&   echo "influxdb soft nofile unlimited" >> /etc/security/limits.conf &&   echo "influxdb hard nofile unlimited" >> /etc/security/limits.conf
 ---> Running in 609b0e0d0adf
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100    31  100    31    0     0     65      0 --:--:-- --:--:-- --:--:--    65
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/usr/lib/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 382, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 3 column 1 (char 29)
```

http://grafanarel.s3.amazonaws.com/latest.json:
```json
{
	"version": "2.0.0-beta3",
}
```

**Comment by hak8or** [comment]
Also getting this error with the JSON being

```JSON
{
	"version": "2.0.0-beta3",
}
```

**Comment by baldwinSPC** [comment]
@archr @hak8or I believe this got fixed in the PR #5
